.circleci/cimodel/data/pytorch_build_data.py

```diff
@@ -13,7 +13,7 @@
         ]),
         ("gcc", [
             ("4.8", [X("3.6")]),
-            ("5.4", [
+            ("5.4", [  # All this subtree rebases to master and then build
```

Comment:
Better to add `compat` to the name of jobs that run the backward compatibility checks? (and then configure the generator script to add a step to merge with master for all `compat` jobs)

Reply:
Yes. I will add bc to the name of test phase, see https://github.com/pytorch/pytorch/pull/26329

This is only for build. Want to share the build stage with some other tests.